service-perfplat/docker/Dockerfile-worker-ci

```diff
@@ -17,10 +17,7 @@ FROM public.ecr.aws/lambda/python:3.8
 
 # fixes security issues detected in build.
 RUN yum update -y rpm openssl curl
```

Comment:
```suggestion
RUN yum update -y rpm openssl curl nss nss-util nss-softtokn nspr
```

Reply:
I thought this would only be a temp, so left it separate. So we don't remove the line entirely when it's removed.